<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
border:2px  solid #00007f;
padding:10px;
background-color: qlineargradient(spread:pad, x1:0.114697, y1:0.103, x2:1, y2:1, stop:0.447761 rgba(5, 70, 207, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>501</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0,0%);
border:none;
background-color: transparent;
border-radios:40px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/centro-recreacional.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 75 12pt &quot;Arial&quot;;
border:none;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>userName</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIngresar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 194, 126);
border-color: rgb(192, 191, 188);
border-color: rgb(222, 221, 218);
color: rgb(0, 0, 0);
font: 75 12pt &quot;Arial&quot;;

QPushButton{
	background-color: rgb(87, 227, 137);

}
QPushButton:hover{
	background-color: rgb(143, 240, 164);

}</string>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>460</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 27, 36);
color: rgb(0, 0, 0);
border-color: rgb(222, 221, 218);
font: 75 12pt &quot;Arial&quot;;

QPushButton{
	background-color: rgb(224, 27, 36);

}
QPushButton:hover{
	background-color: rgb(237, 51, 59);

}</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelError">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUserName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 255);
color: rgb(36, 31, 49);
color: rgb(61, 56, 70);
border-radius: 10px;
border:none;</string>
    </property>
    <property name="placeholderText">
     <string>Ingresa tu nombre de usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="btnError">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(224, 27, 36);
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
